test/common/testlib.py

```diff
@@ -610,6 +621,7 @@ def login_and_go(self, path=None, user=None, host=None, superuser=True, urlroot=
         self.expect_load()
         self._wait_present('#content')
         self.wait_visible('#content')
+        self.adjust_window_for_fixed_content_size()
```

Comment:
We have a few more tests which do custom logins. These would only need to call this if they do pixel tests, right?

Reply:
Yes, and only if they call assert_pixels on their `body` or something else that is affected by the iframe size.